packages/mobile/ios/Podfile.lock

```diff
@@ -812,7 +812,7 @@ SPEC CHECKSUMS:
   FirebaseMessaging: 089b7a4991425783384acc8bcefcd78c0af913bd
   FirebaseStorage: 93fe2f8190a01bfb2b2c4932df7d679744c4ef1d
   Folly: b73c3869541e86821df3c387eb0af5f65addfab4
-  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
+  glog: 1f3da668190260b06b429bb211bfbee5cd790c28
```

Comment:
Looks like these shouldn't be changed. CircleCI complains https://app.circleci.com/pipelines/github/celo-org/celo-monorepo/30707/workflows/95dd5695-2600-42f4-a85c-b0c7ceadbd1b/jobs/376203

Reply:
Do you know why this happens? I find this occurring very frequently for me and I am not sure how to resolve it locally.

Reply:
The Podspec content for `glog` is different between your machine and the CI machines.
Updating your local podspecs should fix it: `bundle exec pod install --repo-update`

Reply:
Unfortunately that doesn't fix it for me